src/components/Dashboard/Tiles/Apps/AppTile.tsx

```diff
@@ -25,7 +26,35 @@ export const AppTile = ({ className, app }: AppTileProps) => {
     .filter((e) => e)
     .join(': ');
 
+  const parsedUrl = useMemo(() => {
+    try {
+      return tldts.parse(window.location.toString());
+    } catch {
+      return null;
+    }
+  }, [window.location]);
+
   const isRow = app.appearance.positionAppName.includes('row');
+  const externalUrl = useMemo(() => {
```

Comment:
This const can contain either app.externalUrl or app.url (which is internal url), so naming it externalUrl seem wrong. Maybe call it href or hrefUrl?
It's semantics though, not important for functionality.